web/src/routes/(user)/sharing/+page.svelte

```diff
@@ -1,10 +1,10 @@
 <script lang="ts">
   import { goto } from '$app/navigation';
-  import empty2Url from '$lib/assets/empty-2.svg';
+  import empty2Url from '@assets/empty-2.svg';
   import AlbumCard from '$lib/components/album-page/album-card.svelte';
   import LinkButton from '$lib/components/elements/buttons/link-button.svelte';
   import UserPageLayout from '$lib/components/layouts/user-page-layout.svelte';
-  import EmptyPlaceholder from '$lib/components/shared-components/empty-placeholder.svelte';
+  import EmptyPlaceHolder from '$lib/components/shared-components/empty-placeholder.svelte';
```

Comment:
Personally, I preferred `EmptyPlaceholder`. Placeholder is just one word, so it's odd to capitalize in the middle. E.g. 300m results for "placeholder" on Google compared to 5m for "place holder". Plus the file is `empty-placeholder.svelte`